### PR TITLE
Issue #1725 update xhyve driver version to 0.3.3 in docs

### LINKED ISSUE #1725: Update xhyve driver version from 0.3.1 to 0.3.3 in the documents
Currently, we mentioned `v0.3.1` xhyve-docker-driver version to docs and it already released to `v0.3.3`, this should need to be updated.

- https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html#xhyve-driver-install

### PR DESCRIPTION
**Pull request by anjannath**
Fix #1725

### CODE REVIEW

**Comment by hferentschik** [comment]
add to whitelist

File: docs/source/getting-started/setting-up-driver-plugin.adoc [hunk 1]
```diff
@@ -107,7 +107,7 @@ $ newgrp libvirt
 [[xhyve-driver-install]]
 == xhyve Driver
 
-{project} is currently tested against *docker-machine-driver-xhyve* version 0.3.1.
+{project} is currently tested against *docker-machine-driver-xhyve* version 0.3.3.
```
Comment (hferentschik) [inline docs/source/getting-started/setting-up-driver-plugin.adoc]: :-)

Comment (praveenkumar) [reply docs/source/getting-started/setting-up-driver-plugin.adoc]: There are other places also where you need to change it. 

=>  You can verify the existing version of the xhyve driver on your system using:

```
$ brew info --installed docker-machine-driver-xhyve
docker-machine-driver-xhyve: stable 0.3.1 (bottled), HEAD  => here also it need to be changed
Docker Machine driver for xhyve
https://github.com/zchee/docker-machine-driver-xhyve
/usr/local/Cellar/docker-machine-driver-xhyve/0.3.1 (3 files, 13.2M) * ==> Same here
  Poured from bottle on 2016-12-20 at 17:44:35
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/docker-machine-driver-xhyve.rb
```

=> Download the docker-machine-driver-xhyve binary using:

```
$ sudo curl -L  https://github.com/zchee/docker-machine-driver-xhyve/releases/download/v0.3.1/docker-machine-driver
```

Comment (anjannath) [reply docs/source/getting-started/setting-up-driver-plugin.adoc]: Made the changes in the two places.

File: docs/source/getting-started/setting-up-driver-plugin.adoc [hunk 2]
```diff
@@ -116,11 +116,11 @@ You can verify the existing version of the xhyve driver on your system using:
 
 ----
 $ brew info --installed docker-machine-driver-xhyve
-docker-machine-driver-xhyve: stable 0.3.1 (bottled), HEAD
+docker-machine-driver-xhyve: stable 0.3.3 (bottled), HEAD
 Docker Machine driver for xhyve
 https://github.com/zchee/docker-machine-driver-xhyve
-/usr/local/Cellar/docker-machine-driver-xhyve/0.3.1 (3 files, 13.2M) *
-  Poured from bottle on 2016-12-20 at 17:44:35
+/usr/local/Cellar/docker-machine-driver-xhyve/0.3.1 (3 files, 10.3MB) *
```
Comment (praveenkumar) [inline docs/source/getting-started/setting-up-driver-plugin.adoc]: @anjannath how come it still have `0.3.1`?

Comment (anjannath) [reply docs/source/getting-started/setting-up-driver-plugin.adoc]: Yikes, fixing it now, i copy pasted the output of `brew` but there was no new lines, then manually edited it, that got left out.

File: docs/source/getting-started/setting-up-driver-plugin.adoc [hunk 3]
```diff
@@ -116,11 +116,11 @@ You can verify the existing version of the xhyve driver on your system using:
 
 ----
 $ brew info --installed docker-machine-driver-xhyve
-docker-machine-driver-xhyve: stable 0.3.1 (bottled), HEAD
+docker-machine-driver-xhyve: stable 0.3.3 (bottled), HEAD
 Docker Machine driver for xhyve
 https://github.com/zchee/docker-machine-driver-xhyve
-/usr/local/Cellar/docker-machine-driver-xhyve/0.3.1 (3 files, 13.2M) *
-  Poured from bottle on 2016-12-20 at 17:44:35
+/usr/local/Cellar/docker-machine-driver-xhyve/0.3.3 (3 files, 10.3MB) *
```
Comment (praveenkumar) [inline docs/source/getting-started/setting-up-driver-plugin.adoc]: 👍

**Pull request merged**